projects/driver_controls/src/power_fsm.c

```diff
@@ -50,15 +58,19 @@ static void prv_state_off(FSM *fsm, const Event *e, void *context) {
   EventArbiterCheck *event_check = fsm->context;
   *event_check = prv_check_off;
 
-  PowerFSMState power_state = POWER_FSM_STATE_OFF;
-
+  PowerFSMState power_state;
   State *current_state = fsm->current_state;
 
-  if (current_state == &state_off_brake) {
-    // No CAN message gets sent
-    return;
+  if (current_state == &state_off) {
+    power_state = POWER_FSM_STATE_OFF;
   } else if (current_state == &state_charging) {
-    power_state = POWER_FSM_STATE_CHARGED;
+    power_state = POWER_FSM_STATE_CHARGING;
+  } else if (current_state == &state_on) {
+    power_state = POWER_FSM_STATE_ON;
+  } else {
+    // state_off_brake and state_charging_brake are simply substates of state_off and
```

Comment:
I like this comment :)

Please document the expected flow (something like Calder's explanation would be great)

Reply:
ping